client/src/main/java/org/spine3/protobuf/Messages.java

```diff
@@ -212,35 +211,47 @@ public static GenericDescriptor getClassDescriptor(Class<? extends Message> claz
         }
     }
 
-    public static Class<?> getFieldClass(FieldDescriptor fieldDescriptor) {
-        final FieldDescriptor.JavaType javaType = fieldDescriptor.getJavaType();
-
-        Object defaultObject = null;
+    /**
+     * Returns the class of the Protobuf message field.
+     *
+     * @param field the field descriptor
+     * @return the class of the field
+     */
+    public static Class<?> getFieldClass(FieldDescriptor field) {
+        final FieldDescriptor.JavaType javaType = field.getJavaType();
         switch (javaType) {
             case INT:
+                return Integer.class;
             case LONG:
+                return Long.class;
             case FLOAT:
+                return Float.class;
             case DOUBLE:
+                return Double.class;
             case BOOLEAN:
+                return Boolean.class;
             case STRING:
+                return String.class;
             case BYTE_STRING:
-                defaultObject = fieldDescriptor.getDefaultValue();
-                return defaultObject.getClass();
+                return ByteString.class;
             case ENUM:
-                //TODO:2016-06-17:alexander.yevsyukov: Figure out how to treat enums.
-                throw Exceptions.unsupported(
-                        "Enums in mapping are not yet supported. Discovered: " + fieldDescriptor.getFullName());
+                final String enumTypeNameStr = field.getEnumType().getFullName();
+                final TypeName enumTypeName = TypeName.of(enumTypeNameStr);
+                return tryConvertToClass(enumTypeName);
```

Comment:
`try` prefix is used to distinguish this method from `public toMessageClass()`. I find it better than `innerToMessageClass()` or something. I use `try` prefix for such methods as Bob Martin does in his great book, `Clean Code`.